### ISSUE TITLE
[Quest][Classic] Too many "Slim's friend"s and they do not stop attacking player after Slim is subdued for quest "The Missing Diplomat (11)"

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5519]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After initiating part 11 of the quest chain from Mikhail, Slim goes stealth and exits the inn. Upon initiating combat with him outside the inn, 3 "Slim's friend"s appear to assist him in the fight when there should only be one Defias spawn appearing. To make things worse, after Slim's health falls below the threshold and he turns friendly, the 3 Defias NPCs remain hostile and continue attacking the player while the patrolling Menethil Harbour guards ignore these Defias NPCs.

**Expected behaviour**: Tell us what should happen instead
There should only be one "Slim's friend" spawning outside the inn to help him. Furthermore, "Slim's friend" should stop attacking once Slim turns friendly. This issue is actually similar to the fight in Stormwind Old Town against Dashel Stonefist in an earlier part of this quest chain.

**Steps to reproduce the problem**:

1. Talk to Mikhail and initiate part 11 of "The Missing Diplomat" quest chain
2. Follow Tapoke "Slim" Jahn as he stealths and exits the inn and initiate combat with him once he turns hostile outside the inn
3. Observe what happens next, in terms of number of hostile Defias NPCs spawned and their subsequent behaviour.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
http://wowwiki.wikia.com/wiki/Quest:The_Missing_Diplomat_(11)

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
